### DISCUSSION TITLE
[@types/selenium-webdriver] Window.fullsceen() should be fullscreen()

### DISCUSSION THREAD

**Discussion by Aelexe** [DefinitelyTyped/DefinitelyTyped#56917] [Issues with a @types package]
Issue located here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/91bd52c1a3c93c6adbe0c20905a9698741159692/types/selenium-webdriver/index.d.ts#L1300

[a comment by an automated account is not shown]

**Comment by 1pete** [comment]
It's a definition of this function, which is under `class Window`, so it's expected

https://github.com/SeleniumHQ/selenium/blob/c7155ed41be1ba36e712524f9b2b130951387117/javascript/node/selenium-webdriver/lib/webdriver.js#L2069-L2073

![image](https://user-images.githubusercontent.com/176037/139779750-2ec914e6-a86c-4d92-ab14-9c76aa7afc23.png)
